hell/adventcalendar/2025/18/index.md

```diff
@@ -1,35 +1,87 @@
 date: 2025-12-18
 author_links:
-  - label: "Site"
-    url: "https://linktoyourblog123.com"
-    link_label: "linktoyourblog123.com"
-intro: "<p>Short introductory text</p>"
+  - label: "Personal website"
+    url: "https://human-centred.nz/"
+    link_label: "human-centred.nz"
+  - label: "Where I work"
+    url: "https://intopia.digital/"
+    link_label: "Intopia"
+intro: "<p>The three most common ways that headings go wonky, and how to fix them!</p>"
 image: "advent25_18"
 ---
-Some text.
-Some text.
 
-Some text. Some text.
 
-## Heading
+We can make our pages easier to understand by using headings to give our pages a clear shape. Our users might visually scan the page, use an extension or bookmarklet, jump around using assistive technology like a screen reader, or ask AI for a summary of the page. High quality headings can make things better for everyone.
 
-Some text.
+In my day job as a Digital Accessibility Consultant, there are a couple of ways that I've seen things go a bit... wonky. Let's go through the three most common ways, and how to fix them.
 
-## Heading
+## Text *should not* be a heading
 
-Some text.
+Ah, this one's a real classic! When we have some big and bold text, for Design Reasons, we sometimes take a bit of a shortcut and mark it up as a heading. Let's say an `<h2>`, because it seems about the right size, or that’s what it says in the design file. But here's the thing: this text doesn't introduce or describe the content that follows. It just "needs" to be big for the look of it. 
 
+This is an issue because when things are marked up as headings that are not headings, it makes the page harder to understand. For example: users of assistive technology like screen readers hear things read as headings of section that are not headings.
 
-<p class="highlight"><strong>Note:</strong> Some text.</p>
+### How to fix it
 
-```html
-<h1> 
-  <a href="/">
-    Hello World
-  </a>
-</h1>
-```
+Stop using HTML and start using CSS. Instead of using a heading element, using a `<p>` or `<span>` or  `<div>` element and use CSS to make it big and bold. Noice.
+
+## Text *should* be a heading
+
+Now let's come from the other side. We look at a design and see some bold text with some big, some bigger, some biggest. Sweet! We fling down a bunch of `<div>` elements, add some styles and we're done. It looks just like the design, chef's kiss, and so on. But here's the thing: this text looks like a heading, but doesn't have any semantics.
+
+This is an issue because when things that are headings aren't marked up as headings, it makes the page harder to understand. For example: screen readers won't announce it as a heading with its level, making it hard to understand the content and structure when listening to all the headings. Excellent.
```

Comment:
```suggestion
This is an issue because when things that are headings aren't marked up as headings, it makes the page harder to understand. For example: screen readers won't announce it as a heading with its level, making it hard to understand the content and structure when listening to all the headings.
```